### DISCUSSION TITLE
I am trying to import a folder to vscode but it wont connect to wsl after importation

### DISCUSSION THREAD

**Discussion by intellygentle** [Cyfrin/foundry-full-course-cu#1014] [Q&A]
There is a code repo I downloaded on github and unzipped the file. I am trying to upload it to vscode and so that I can use it with Foundry but after uploading it, the terminal is just a powershell not the one we use for foundry.

**Comment by 0xusmanf** [comment]
WSL is a separate virtual environment, it can not interact with any folder saved on your Windows directories. You can follow the instructions below to open that folder in WSL.

Step-1 Open the VS code and click on the green button on the bottom left corner

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/110072406/0fdb70a5-e97f-4d01-a126-34df85436488)

Step-2 Click on Connect to WSL

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/110072406/f777995b-b1a2-414e-8628-5fc803449a92)

It will look like this after it's connected.

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/110072406/f13ca10b-6f5c-4861-a4eb-9e946e21ffbb)

Wait for it to connect, once it connects you have 2 options.

Option-1 Clone the repo directly through terminal by `git clone REPO_LINK .`, make sure you have git installed by running `git --version` command in your WSL terminal. You can follow the instructions [here](https://www.atlassian.com/git/tutorials/install-git#linux) to install git on Linux.

Option-2 Head over to your WSL root directory in your file explorer and manually copy past the folder there, for example in my case the WSL root directory is `\\wsl.localhost\Ubuntu\home\usmanfarooq90`.

**Reply by intellygentle** [reply]
Done! Thanks!
